### PR TITLE
Add loading and saving of requests

### PR DESCRIPTION
**Pull request by esphen**
This PR adds a collections tab to the request pane that has a user's saved requests. A user can click "Save request" when a request has been entered, and it is persisted to IndexedDB.

Also adds the following:

  - Fixed broken master build
  - Prefixes URLs with http:// when request is submitted so requests are actually sent when it is missing
  - Continued work on Nikolay Damyanov (@set-killer)s indexedDB implementation. Most of the glory goes to him for laying the foundations of the work on the collections tab
  - Adds localization for new features with the help of Google Translate.. So please have a glance at this and comment if you know the languages!

### CODE REVIEW

**Comment by set-killer** [comment]
Thank you @esphen for the implementation of this feature. You are awesome!
Anyway, no glory for me. I did nothing else but brokin' up the master repository.

**Comment by nathan-osman** [comment]
Thanks to both of you!

File: locale/ro-RO/resteasy.properties [hunk 1]
```diff
@@ -12,6 +12,11 @@ header.about=About
 highlight.notice=Răspunsul este mai mare de 10 KiB. Evidențierea sintaxei poate dura o perioadă lungă de timp.
 highlight.highlight=Evidențiază oricum
 request.send=Lansează
+request.save=Salvare cerere
```
Comment (revolter) [inline locale/ro-RO/resteasy.properties]: Salvează cererea

File: locale/ro-RO/resteasy.properties [hunk 2]
```diff
@@ -12,6 +12,11 @@ header.about=About
...
+request.tab.request=Cerere
+request.tab.collections=Colecții
+request.collections.title=Colecții
+request.collections.noCollections=Nu există colecții. Adauga apăsând salva pe cerere.
```
Comment (revolter) [inline locale/ro-RO/resteasy.properties]: Nu există colecții. Adaugă apăsând salvează pe o cerere.

Comment (esphen) [inline locale/ro-RO/resteasy.properties]: Thanks @revolter! You want to make the changes real quick @nathan-osman, or shall I submit another PR?

**Comment by nathan-osman** [comment]
If you could submit another PR quick, that would be great.

**Comment by esphen** [comment]
Done!

**Comment by revolter** [comment]
My pleasure @esphen!
